content/reference/endpoints/change_overs.md

```diff
@@ -0,0 +1,43 @@
+# ChangeOvers
+
+1. TOC
+{:toc}
+
+### Parameters
+<ul class="nav nav-pills" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | ChangeOver's id.
+-----------------|---------|------------|------------
+map              | String  | Read       | ChangeOver's map of booking status. There is a total of 1096 characters, each representing one day. List of statuses described in [enums section](/reference/enums#change_over-map-statuses).
+-----------------|---------|------------|------------
+start_date       | [Date](/reference/enums#formats) | Read       | ChangeOver's start date.
+updated_at       | [Time](/reference/enums#formats) | Read       | ChangeOver's update time.
+{: class="table table-bordered"}
+  </div>
+</div>
+
+## List change_overs
+
+List all change_overs for given account.
```

Comment:
`for a given account` #grammarNazi

Reply:
There is no `a` in all endpoint 🐶 Are you sure ?

Reply:
Well, it means it's wrong everywhere :D